src/Microsoft.Health.Dicom.Core/Exceptions/ElementValidationExceptions.cs

```diff
@@ -0,0 +1,100 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Globalization;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Validation;
+
+namespace Microsoft.Health.Dicom.Core.Exceptions
+{
+    internal static class ElementValidationExceptions
```

Comment:
nit: these are all methods, so they should probably start with a verb, like `CreateDateIsInvalidException`. If we do something like this, then maybe this class name should end with "Factory," like `ValidationExceptionFactory`?